app/src/main/java/com/eveningoutpost/dexdrip/ui/MicroStatus.java

```diff
@@ -12,4 +12,6 @@ public interface MicroStatus {
 
     boolean xmitterBattery();
 
+    boolean using_g7(); // True when using G7
```

Comment:
not java style method naming. Java style method naming is camel case
also use java doc for "True when using G7"